tests/conftest.py

```diff
@@ -0,0 +1,114 @@
+import jinja2
+import logging
+from OpenSSL import crypto
+import os
+import pytest
+import subprocess
+import time
+
+
+pytest_plugins = [
+    "mock_fixtures",
+]
+
+
+def pytest_configure():
+    pytest.module_capsys = None
```

Comment:
Is `pytest.module_capsys.disabled` below not enough?

BTW, I was not able to find "module_capsys" in docs, so it might worth a comment.

Reply:
This is a hack. `pytest.module_capsys` is just a global var.  You need some global var to keep the fixture object.

Reply:
Does it have to be global variable in 3rd party module namespace?

Anyway, I think, we should just direct log output to stderr and remove "showlog" option, as pytest already provides various ways to show captured output (e.g. "--capture"). We still need to keep file for the sake of alerts, so nginx config will contain both, i.e.
```
    error_log stderr info;
    error_log {testdir}/error.log info;
```

Also, "info" level might be more appropriate, as, for example, Nginx logs malformed HTTP requests at that level.

Reply:
Fixed